docs/design/storagenode-downtime-tracking.md

```diff
@@ -50,9 +48,9 @@ The current Satellite database has the table `nodes`. For the offline time calcu
 
 ### Detecting offline nodes
 
-Per [Kademlia Removal design document](https://github.com/storj/storj/blob/master/docs/design/kademlia-removal.md#network-refreshing), any Storage Node has to ping the Satellite every hour and when it starts up. We have to contact the Storage Nodes which the Satellite hasn't registered a ping for more than one hour to determinie if they are offline or for whatever reason their ping didn't reach the Satellite.
+Per [Kademlia removal design document](https://github.com/storj/storj/blob/master/docs/design/kademlia-removal.md#network-refreshing), any storage node has to ping the satellite every hour. For storage nodes that have not pinged, we need to contact them directly.
```

Comment:
This was an addition requested by @mobyvb ; for me, it wasn't needed to be mentioned because it's already commented that this document is based on those requirements.

If this simplification is fine for him, it's fine for me too.